extensions/positron-catalog-explorer/src/catalogs/databricks.ts

```diff
@@ -113,6 +114,24 @@ async function registerDatabricksCatalog(
 	if (!token) {
 		return;
 	}
+
+	// Validate the token by checking if the user is available
+	try {
+		const userResponse = await fetch(`${workspace}/api/2.0/workspace/list?path=/`, {
```

Comment:
Is there a chance this could be an expensive call? Not sure if we can do anything about it if so, but I'm just wondering. Is there a page size parameter or something?